packages/aws-cdk-lib/pipelines/lib/codepipeline/codepipeline-source.ts

```diff
@@ -425,11 +425,11 @@ class CodeStarConnectionSource extends CodePipelineSource {
     super(`${repoString}-${branch}`);
 
     const parts = repoString.split('/');
-    if (Token.isUnresolved(repoString) || parts.length !== 2) {
+    if (Token.isUnresolved(repoString) || parts.length < 2) {
       throw new Error(`CodeStar repository name should be a resolved string like '<owner>/<repo>', got '${repoString}'`);
```

Comment:
Same thing here.

Reply:
We might also consider changing the verbiage in the error directory regarding the expected resolved string format.  As an example, the same repository name in the console expresses the format as group/subgroup/project.